src/Common/ErrorParserUtility.ts

```diff
@@ -1,67 +1,69 @@
-﻿import * as DataModels from "../Contracts/DataModels";
-import * as ViewModels from "../Contracts/ViewModels";
-
-export function replaceKnownError(err: string): string {
-  if (
-    window.dataExplorer.subscriptionType() === ViewModels.SubscriptionType.Internal &&
-    err.indexOf("SharedOffer is Disabled for your account") >= 0
-  ) {
-    return "Database throughput is not supported for internal subscriptions.";
-  }
-
-  return err;
-}
-
-export function parse(err: any): DataModels.ErrorDataModel[] {
```

Comment:
I did not remove ErrorParserUtility from eslintignore because of several uses of `any` for errors. I don't think we're using any unified error object (this fact spurred the investigation into this fix in the first place). I think this will entail a more thorough fix throughout several files.